### ISSUE TITLE
Installer auto restarts computer

### ISSUE THREAD

**Issue by Jesper-Hustad** [MCJack123/craftos2#151]
**Describe the bug**
After installer finished it automatically restarted my computer.  
I was able to cancel it, the install wizard still had "Do you wish to restart now?", but this is after it tried to restart machine.

**To Reproduce**
1. Go to https://www.craftos-pc.cc/  
2. Click on Download for Windows  
3. Ignore Windows Defender virus warning and run anyway  
3. Follow setup wizard  
4. Machine tries to restart without consent  

**Expected behavior**
User can choose whether they want to restart now or later

**Environment:**
 - OS: Windows  
 - Version: Build 18363  
 - Compiled from source? [No]

**Additional context**
Is Windows Defender warning expected behavior? Could be good to inform user if this is the case?  
Maybe this is a known bug from the setup wizard or some configuration is wrong?

**Comment by MCJack123** [comment]
Argh, this bug is annoying. I've tried to fix it multiple times, but it always reboots anyway. It seems to be something with Inno Setup, so I'll see if there's any known issues there that are related.

In addition, the SmartScreen warning is there because the executables aren't signed. EV Code Signing certificates are hundreds of dollars, even surpassing Apple's $100/year developer fee. I'm not about to spend a whole bunch of money just to make a warning pop-up disappear for the one Windows program I maintain. I'll add a note about it in the docs, though.

**Comment by Jesper-Hustad** [comment]
No problem, i don't expect expect you to spend money on any signed certificates of course!

From what i read about Inno Setup it [should not be possible](https://stackoverflow.com/a/27277351/11590809) for this to happen, what a weird bug. 

I'm guessing you create install wizards for the craftos-pc.cc site, my best guess is probably a weird version or config?

Anyways thanks for the response!

**Comment by MCJack123** [comment]
So I did a bit more digging, and it appears the problem may be in the Visual C++ Redistributable. Apparently, there are some cases in which it can end up triggering a restart without a prompt. I've added `/norestart` to the command line flags in the Inno source file, but I haven't gotten a chance to build a full installer yet to test this. A new update is coming soon (to prepare for v2.5 next month), so I'll see if the rebooting is still happening then.

One thing I've noticed is that only Windows 10 appears to be affected by this issue. My Windows 7 VM that I've used to test on hasn't even prompted for a reboot at all - which is surprising, since MSVC ships an update for Vista/7/8.1 which can require a reboot.
